benchmarks/lucet-benchmarks/src/par.rs

```diff
@@ -42,7 +42,7 @@ fn par_instantiate<R: RegionCreate + 'static>(c: &mut Criterion) {
     let workdir = TempDir::new().expect("create working directory");
 
     let so_file = workdir.path().join("out.so");
-    compile_hello(&so_file, OptLevel::Fast);
+    compile_hello(&so_file, OptLevel::default());
```

Comment:
Could you make these be `SpeedAndSize` explicitly for the benchmark suite, in case the default changes?

Reply:
good idea! i put a `const BENCHMARK_OPT_LEVEL` in each source file that uses it in benchmarks.